packages/react-charting/package.json

```diff
@@ -31,7 +31,8 @@
     "@fluentui/react": "^8.91.1",
     "@types/react-addons-test-utils": "0.14.18",
     "@fluentui/scripts": "^1.0.0",
-    "@fluentui/jest-serializer-merge-styles": "^8.0.20"
+    "@fluentui/jest-serializer-merge-styles": "^8.0.20",
+    "enzyme-to-json": "3.6.2"
```

Comment:
can you please move this to root `package.json` devDependency   ? we are in the process to adhere to single version policy approach for devDependencies, thus this violates that. ty

Reply:
done